README.md

```diff
@@ -1,8 +1,12 @@
 # ansible-inclusion
-Requests to include new collections into the ansible package
+Requests to include new collections into the ansible package.
 
+All new collections submitted for inclusion must satisfy the [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst).
+If there are any questions, feel free to ask for clarifications in the discussion described below and / or in the `#ansible-community` IRC channel.
 
 Add a new Discussion with the Category: `New collection review`.  Please include the following information:
 
 * Link to the galaxy page for the collection
-* Any github ids that should be notified if there's an issue with the collection and the bug tracker has not yielded information
+* Any github IDs that should be notified if there is an issue with the collection and the bug tracker has not yielded information
+* Re-check that your collection satisfies the [Collection Requirements](https://github.com/ansible-collections/overview/blob/main/collection_requirements.rst)
+* Put a comment "The collection satisfies all the requirements and is ready for review" in the discussion.
```

Comment:
```suggestion
* Put a comment "The collection satisfies all the requirements and is ready for review" in the discussion
```

Reply:
also removed a period

Reply:
(Sorry I've put periods everywhere because the sentences start with a capital letter)